openedx/core/djangoapps/monitoring_utils/middleware.py

```diff
@@ -114,7 +114,10 @@ def _log_prefix(self, prefix, request):
         """
         Returns a formatted prefix for logging for the given request.
         """
-        return u"{} request '{} {} {}'".format(prefix, request.method, request.path, self._cache[self.guid_key])
+        # During some operations (like during the course export polling), process_response will be executed
+        # with no corresponding process_request call.
```

Comment:
Per our discussion, the real underlying issue is with the request_cache being cleared out after processing a celery task in-process: https://github.com/edx/edx-platform/blob/master/common/djangoapps/request_cache/__init__.py#L28